src/app/components/gallery/home/gallery.component.ts

```diff
@@ -85,8 +107,10 @@ export class GalleryComponent implements OnInit, OnDestroy {
 
     @autobind()
     public openReadme(applicationId: string) {
-        const link = `https://github.com/Azure/BatchExplorer-data/tree/master/ncj/${applicationId}`;
-        this.electronShell.openExternal(link, { activate: true });
+        this._baseUrl.subscribe((baseUrl) => {
```

Comment:
you should do this._baseUrl.pipe(take(1)).subscribe or it will keep opening a new tab here if the settings changes

Reply:
you can also save the baseUrl as a string instead of observable too by subscribing in the constructor